### PR TITLE
Feat-database-insertmethods

### PR DESCRIPTION
**Pull request by ryanakiyama**

### CODE REVIEW

**Comment by ryanakiyama** [comment]
Note: Will refactor database methods to use promises

File: schema.sql [hunk 1]
```diff
@@ -65,5 +65,17 @@ CREATE TABLE availability (
...
+INSERT INTO destinations (destination_name) VALUES ("San Francisco");
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Glen Park", 1);
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Golden Gate Park", 1);
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Haight Ashbury", 1);
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Inner Richmond", 1);
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Inner Sunset", 1);
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Outer Richmond", 1);
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Outer Sunset", 1);
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Presidio", 1);
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Financial District", 1);
```
Comment (tiffaphm) [inline schema.sql:77]: Do you plan to add more neighborhoods post-mvp? Or were these the most popular neighborhoods in the current market?

File: database/index.js [hunk 2]
```diff
@@ -6,3 +6,48 @@ var connection = mysql.createConnection({
...
+            if (err) {
+              callback(err, null);
+            } else {
+              console.log(destinationsId, neighborhoodsId, hostsId);
+              connection.query('INSERT into listings (listing_name, rating, rating_count, room_type, hosts_id, destinations_id, neighborhoods_id) VALUES (?, ?, ?, ?, ?, ?, ?)', [listings.listing_name, listings.rating, listings.number_of_ratings, listings.room_type, `${hostsId[0]['id']}`, `${destinationsId[0]['id']}`, `${neighborhoodsId[0]['id']}`], function (err, results, fields) {
+                if (err) {
+                  callback(err, null);
+                } else {
+                  console.log(listings.room_type);
+                  callback(null, results);
```
Comment (tiffaphm) [inline database/index.js:29]: Make sure you properly test these callbacks since there are so many of them! ;)

File: server/index.js [hunk 3]
```diff
@@ -5,7 +5,31 @@ const dataGenerator = require('./data-generator');
 const db = require('../database/index.js');
 app.use(bodyParser.json());
 
-app.get('/reservations', function (req, res) {
+app.post('/listings', function (req, res) {
+  var listings = dataGenerator.listingsGenerator();
```
Comment (tiffaphm) [inline server/index.js]: change all 'var' declaration to use ES6

File: server/data-generator.js [hunk 4]
```diff
@@ -15,37 +15,40 @@ const neighborhoods = [
 
 const listingsGenerator = function() {
   const listings = [];
-  for (var i = 0; i < 3; i++) {
+  for (var i = 0; i < 50000; i++) {
```
Comment (tiffaphm) [inline server/data-generator.js]: make changes to 'var'

File: server/data-generator.js [hunk 5]
```diff
@@ -15,37 +15,40 @@ const neighborhoods = [
...
                   };  
   }
   return listings;
 }
 
-module.exports.listingsGenerator = listingsGenerator;
\ No newline at end of file
+const availabilityGenerator = function() {
+  const availability = [];
+  for (var i = 0; i < 50000; i++) {
```
Comment (tiffaphm) [inline server/data-generator.js]: make changes to 'var', also think about using ES6 arrow functions

**Review by tiffaphm — CHANGES_REQUESTED** [review]
It might be worth going through these files again and changing some syntax to use ES6 - make sure your linter is working! :)

File: schema.sql (hunk 1 above)
Comment (ryanakiyama) [reply schema.sql:77]: Planning to add more neighborhoods post mvp

File: database/index.js (hunk 2 above)
Comment (ryanakiyama) [reply database/index.js:29]: FIxed!

**Review by tiffaphm — APPROVED** [review]

**Pull request merged**